src/lib/y2storage/planned_volumes_list.rb

```diff
@@ -23,6 +23,8 @@
 
 require "yast"
 require "y2storage/planned_volume"
+require "y2storage/proposal/proposed_partition"
+require "y2storage/proposal/proposed_lv"
```

Comment:
I don't see a reason for these two requires. The rest of the diff for this class is all red, so just removing code, no new code using these classes.